tests/postgres/alltypes_test.go

```diff
@@ -2,6 +2,7 @@ package postgres
 
 import (
 	"database/sql"
+	"github.com/lib/pq"
 	"testing"
 	"time"
 
```

Comment:
As for testing, maybe we can have a new array only table, similar to what's already been done for range tests.

Reply:
Note that test files are in separate repo: https://github.com/go-jet/jet-test-data